cookie.js

```diff
@@ -223,3 +215,13 @@ function tryDecode (str, decode) {
     return str
   }
 }
+
+/**
+ * Module exports.
+ * @public
+ */
+
+module.exports = {
+  parse,
+  serialize
+}
```

Comment:
please restore the original setup, that works with ESM while this does not.

Reply:
Yeah but it’s a named export from the main file

So you ideally import everything from there